src/clj/game/core/costs.clj

```diff
@@ -21,6 +21,9 @@
     [game.utils :refer [quantify]]
     [clojure.string :as string]))
 
+;; Zero stealth value for costs where it doesn't make sense
+(defmethod stealth-value :default [_] nil)
```

Comment:
I'd prefer this to be directly under the `defmethod value` below. Keeps everything "together" and matches how you have it in `payment.clj`

Reply:
```suggestion
(defmethod stealth-value :default [_] 0)
```